pkg/resource/graph/dependency_graph.go

```diff
@@ -279,6 +279,14 @@ func (dg *DependencyGraph) TransitiveDependenciesOf(r *resource.State) mapset.Se
 	return dependencies
 }
 
+func (dg *DependencyGraph) ChildrenOf(res *resource.State) []*resource.State {
```

Comment:
Even though this is WIP, I would document this function in this PR, since it's a "library" function and feels unlikely to change?

Reply:
:heavy_check_mark: